src/main/java/com/project/model/UserRoles.java

```diff
@@ -0,0 +1,6 @@
+package com.project.model;
+
+public enum UserRoles {
```

Comment:
Your enum should be named `UserRole` since when you use enum you take a singular value for instance `ADMIN`.

Reply:
renamed enumeration to `UserRole`